GVFS/GVFS.Installer.Mac/vfsforgit_components.plist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<array>
+	<dict>
+		<key>BundleHasStrictIdentifier</key>
+		<true/>
+		<key>BundleIsRelocatable</key>
+		<false/>
+		<key>BundleIsVersionChecked</key>
```

Comment:
How do we set the version of the bundle?

Reply:
This might be the key to understanding why we suddenly got hit with this issue, we may not be versioning it correctly today. Either way, I believe this is still a correct change to make.

Reply:
> This might be the key to understanding why we suddenly got hit with this issue

We started seeing this shortly after we included the notification app in the installer :)

Reply:
>> How do we set the version of the bundle?

Right now we are not updating the Bundle version (it is set to 1.0). I will take a look at the options (scripts used to set the ProjFS kext bundle version & also Apple's versioning tool `agvtool`) and incorporate version updating into the build script. Updated the [TODO](https://github.com/microsoft/VFSForGit/issues/1262) issue.